src/common/ceph_context.cc

```diff
@@ -182,6 +182,7 @@ class MempoolObs : public md_config_obs_t,
 
   // AdminSocketHook
   int call(std::string_view command, const cmdmap_t& cmdmap,
+	   const bufferlist& inbl,
```

Comment:
How about not naming the parameter in the implementations that don't make use of it? `[[maybe_unused]]` would be an alternative.